java_io_block/main_task/src/main/java/IoMainTaskMain.java

```diff
@@ -0,0 +1,39 @@
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Collections;
+import java.util.List;
+
+public class IoMainTaskMain {
+    public static void main(String[] args) throws IOException {
+        // You should input the absolute path to dir or txt file
+        //For example you can use absolute path to java_io_block/main_task/data or java_io_block/main_task/input.txt
+
+        BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
+        String input = reader.readLine();
+
+        if (input.contains(".txt")) {
```

Comment:
I can create directory `my.dir.txt.test1` and it will pass this check

I suggest `.endsWith` && `!isDirectory`

Reply:
done